sklearn/preprocessing/tests/test_discretization.py

```diff
@@ -0,0 +1,230 @@
+from __future__ import absolute_import
+
+import numpy as np
+import scipy.sparse as sp
+import warnings
+
+from sklearn.externals.six.moves import xrange as range
+from sklearn.preprocessing import KBinsDiscretizer
+from sklearn.preprocessing import OneHotEncoder
+from sklearn.utils.testing import (
+    assert_array_equal,
+    assert_raises,
+    assert_raise_message,
+    assert_warns_message
+)
+
+X = np.array([[-2, 1.5, -4, -1],
```

Comment:
Nit: I think this could be a nested list, and doesn't have to be `np.array`.

Reply:
I think format invariance is handled in common tests.